src/loaders/patchNextClientPageLoader.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+
+/**
+ * This webpack loader patches next/dist/client/page-loader.js file.
+ * Also it requires `include-defaults.js` with required shared libs
+ *
+ * @type {(this: import("webpack").LoaderContext<{}>, content: string) => string>}
+ */
+function patchNextClientPageLoader(content) {
+  if (content.includes('MFClient')) {
+    // If MFClient already applied then skip patch
+    return content;
+  }
+
+  // avoid absolute paths as they break hashing when the root for the project is moved
+  // @see https://webpack.js.org/contribute/writing-a-loader/#absolute-paths
+  const pathIncludeDefaults = path.relative(
+    this.context,
+    path.resolve(__dirname, '../include-defaults.js')
+  );
+  path.resolve(__dirname, '../include-defaults.js');
+  const pathMFClient = path.relative(
+    this.context,
+    path.resolve(__dirname, '../client/MFClient.js')
+  );
+
+  patchedContent = content.replace(
+    'exports.default = PageLoader;',
+    `
+      require("${pathIncludeDefaults}");
```

Comment:
require('${JSON.stringify(pathIncludeDefaults)}') might be better bet

Reply:
Yep, you are right. Without it, windows' users might be affected with back-slashes.
Fixed https://github.com/module-federation/nextjs-mf/pull/103/commits/8178b19131b1648a92748dea9cc23dd9eae81f17